lib/model/autocomplete.dart

```diff
@@ -148,6 +148,14 @@ class AutocompleteViewManager {
     autocompleteDataCache.invalidateUser(event.userId);
   }
 
+  void handleMessageEvent(MessageEvent event) {
+    _refreshStaleUserResults();
+  }
+
+  void handleOlderMessages() {
+    _refreshStaleUserResults();
+  }
```

Comment:
> …ah, I guess this will be used when we start considering the current topic/stream, which I see you're working on in https://github.com/zulip/zulip-flutter/pull/692. Makes sense.

I removed it for this PR and will add it to the next one where it is used. Firstly, I thought this was necessary for `RecentDmConversationsView.latestMessagesByRecipient`.